beaker.gemspec

```diff
@@ -40,6 +40,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'inifile', '~> 2.0'
   s.add_runtime_dependency 'rsync', '~> 1.0.9'
   s.add_runtime_dependency 'open_uri_redirections', '~> 0.2.1'
+  s.add_runtime_dependency 'in-parallel', '~> 0.1.8'
```

Comment:
The way the dependency is set now, beaker will not automatically get the `0.2` release, it will only get the latest `0.1.x` release. By specifying `~> 0.1` it will install the latest available gem in the range of `> 0.1.0` to `< 1.0`.

See this article for more pessimistic operator explaining: https://robots.thoughtbot.com/rubys-pessimistic-operator